src/Transport/obsoletes-v3.cs

```diff
@@ -22,6 +23,12 @@ public static partial class AzureServiceBusTransportSettingsExtensions
             this TransportExtensions<AzureServiceBusTransport> transportExtensions,
             Func<string, string> subscriptionNameShortener)
             => throw new NotImplementedException();
+
+        [ObsoleteEx(ReplacementTypeOrMember = "CustomTokenCredential(string fullyQualifiedNamespace, TokenCredential tokenCredential)",
+            TreatAsErrorFromVersion = "3",
+            RemoveInVersion = "4")]
+        public static TransportExtensions<AzureServiceBusTransport> CustomTokenCredential(this TransportExtensions<AzureServiceBusTransport> transportExtensions, TokenCredential tokenCredential)
```

Comment:
Why is this obsolete? Doesn't this mean there's no way to use a `TokenCredential` in the not obsolete `Transport<T>` API?

Reply:
That's what I get for not pulling this down and looking at it locally. I find the GitHub UI for PR reviews to be rather terrible!